datahub-web-react/src/app/entity/shared/containers/profile/sidebar/AboutSection/LinksSection.tsx

```diff
@@ -0,0 +1,34 @@
+import React from 'react';
+import styled from 'styled-components';
+import LinkButton from '../LinkButton';
+import { useEntityData, useRefetch } from '../../../../EntityContext';
+import { AddLinkModal } from '../../../../components/styled/AddLinkModal';
+
+const AddLinksWrapper = styled.div`
+    margin-left: -15px;
+`;
+
+interface Props {
+    hideLinksButton?: boolean;
+    readOnly?: boolean;
+}
+
+export default function LinksSection({ hideLinksButton, readOnly }: Props) {
```

Comment:
Is this just a file being moved?

Reply:
yup! just pulled it out from the old bigger SidebarAboutSection component.

Reply:
nice!